extensions/amp-analytics/0.1/vendors.js

```diff
@@ -349,7 +349,41 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
       'image': true,
     },
   },
-
+	
+'catchpoint': {
+	
+    'requests': {	
```

Comment:
I know you added test in #16420. But that's not going to work. One of our tests test that every vendor has a related test. In order to pass this test and get this PR in, you will need to move the change in #16420 to this PR. Travis is not complaining now because it terminates early because of the lint errors.